app/src/main/res/values/strings.xml

```diff
@@ -483,6 +483,10 @@
          from being restarted. -->
     <string name="restart_later_dialog_button">Restart Later</string>
 
+    <!-- This string represents the back button the user presses in order to cancel the selection
+         that required restart. -->
+    <string name="restart_back_dialog_button">Back</string>
```

Comment:
Since this will not be shown to the user, maybe we can just use the existing `"back_button"` string so we don't need to add a new one?